sdf/cams.go

```diff
@@ -255,7 +255,11 @@ func MakeThreeArcCam(
 
 	// distance between base and nose circles
 	distance := baseRadius + lift - noseRadius
-	return ThreeArcCam2D(distance, baseRadius, noseRadius, flankRadius)
+	tac2d, err := ThreeArcCam2D(distance, baseRadius, noseRadius, flankRadius)
+	if err != nil {
+		return nil, err
+	}
+	return tac2d, nil
 }
```

Comment:
ThreeArcCam2D() returns sdf, error, the same as MakeThreeArcCam() so this code does nothing.

Reply:
Yep -- good catch.  That was one of the last ones we reviewed at the end of a 4 hour session; fatigue had set in so it didn't jump out at me.

Reply:
- reverted in 2fb91c2